hl/src/H5LTanalyze.c

```diff
@@ -78,9 +78,11 @@
 #ifndef FLEXINT_H
 #define FLEXINT_H
 
-/* C99 systems have <inttypes.h>. Non-C99 systems may or may not. */
+/* C99 systems have <inttypes.h>. Non-C99 systems may or may not.
+ * C11 removed __STDC_LIMIT_MACROS.
+ */
 
-#if defined (__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
+#if defined (__STDC_VERSION__) && (__STDC_VERSION__ >= 199901L) && (__STDC_VERSION__ < 201112L)
 
```

Comment:
This is a generated file so it should not be edited.

Reply:
Ah yes, will remove.  I wonder if newer flex fixes this...